qemu/tests/cfg/block_during_io.cfg

```diff
@@ -48,7 +48,7 @@
     variants:
         - iozone_stress:
             stress_name = iozone
-            iozone_cmd_opitons = "-azR -r 64k -n 1G -g 4G -M -i 0 -i 1 -I "
+            iozone_cmd_opitons = "-azR -r 64k -n 1G -g 10G -M -i 0 -i 1 -I "
```

Comment:
Hi @menli820 , With the 10G test meet some other issues before, sometimes system disk will be full of written, then cause all cases behind this one failed.

Reply:
Here we can try to let the iozone send the random io and smaller block size, like 512b to extend the io time.

Reply:
Refer to https://www.iozone.org/docs/IOzone_msword_98.pdf
or use the -a option
```
-a
Used to select full automatic mode. Produces output that covers all tested file operations
for record sizes of 4k to 16M for file sizes of 64k to 512M.
```

Reply:
Thanks all !
Try iozone to send smaller block size 512b here.

```
 (1/6) Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: STARTED
 (1/6) Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (718.87 s)
 (2/6) repeat1.Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_during_io.iozone_stress.system_disk.send_qmp.with_system_reset.q35: STARTED
 (2/6) repeat1.Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_during_io.iozone_stress.system_disk.send_qmp.with_system_reset.q35: PASS (285.24 s)
 (3/6) repeat2.Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_during_io.iozone_stress.system_disk.send_qmp.with_system_reset.q35: STARTED
 (3/6) repeat2.Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_during_io.iozone_stress.system_disk.send_qmp.with_system_reset.q35: PASS (270.89 s)
 (4/6) repeat3.Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_during_io.iozone_stress.system_disk.send_qmp.with_system_reset.q35: STARTED
 (4/6) repeat3.Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_during_io.iozone_stress.system_disk.send_qmp.with_system_reset.q35: PASS (267.33 s)
 (5/6) repeat4.Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_during_io.iozone_stress.system_disk.send_qmp.with_system_reset.q35: STARTED
 (5/6) repeat4.Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_during_io.iozone_stress.system_disk.send_qmp.with_system_reset.q35: PASS (264.34 s)
 (6/6) repeat5.Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_during_io.iozone_stress.system_disk.send_qmp.with_system_reset.q35: STARTED
 (6/6) repeat5.Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_during_io.iozone_stress.system_disk.send_qmp.with_system_reset.q35: PASS (261.65 s
```